Fix getting control instance (#337)
tarantool/ansible-cartridge: CHANGELOG.md

README.md to use the newest tag with new release
-->

### Fixed

> - Fail on getting control instance when there are not unjoined instances with
>   `replicaset_alias` set

Reviewer: ```suggestion
- Fail on getting control instance when there are only unjoined instances without `replicaset_alias`
```
